src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -245,26 +266,38 @@ const StackedBarChart = createReactClass({
       }
     });
     return title;
-  },
+  };
 
-  renderChartColumn(point, maxval, pointWidth) {
+  getMinHeight(index, pointLength) {
+    let {minHeights} = this.props;
+    return minHeights && (minHeights[index] || minHeights[index] == 0)
+      ? this.props.minHeights[index]
+      : 1;
+  }
+
+  renderChartColumn(point, maxval, pointWidth, index, totalPoints) {
     let totalY = point.y.reduce((a, b) => a + b);
     let totalPct = totalY / maxval;
     let prevPct = 0;
     let pts = point.y.map((y, i) => {
-      let pct = totalY && this.floatFormat(y / totalY * totalPct * 99, 2);
+      let pct = Math.max(
+        totalY && this.floatFormat(y / totalY * totalPct * 99, 2),
```

Comment:
(takes a long, deep breath...)

As best I can tell the "99" is used to reserve a little extra space for the `min-heights` that were previously applied to bars with css (now applied with the prop). this didn't *really* work because 1% isn't always 1px and sometimes (like with issue release charts) the min-height is 4+ pixels. 

However none of the charts have overflow: hidden so if the bars exceed 100% height it doesn't matter (so long as visually it doesn't look weird)

I kept the 99 in there to keep things similar to what they were before. Truthfully, the charts have never been that accurate lol. We could do 100 and nobody would notice.

Reply:
Also good for this to be in commentz